server/src/main/java/org/opensearch/search/profile/query/CollectorResult.java

```diff
@@ -113,6 +163,19 @@ public CollectorResult(StreamInput in) throws IOException {
             CollectorResult child = new CollectorResult(in);
             this.children.add(child);
         }
+        if (in.getVersion().onOrAfter(Version.V_3_0_0)) {
+            this.reduceTime = in.readLong();
+            this.maxSliceTime = in.readLong();
+            this.minSliceTime = in.readLong();
+            this.avgSliceTime = in.readLong();
+            this.sliceCount = in.readVInt();
+        } else {
+            this.reduceTime = 0L;
```

Comment:
Just a thought here, since you set `reduceTime` & others to `time` (if not available) and `sliceCount` to 1,  should we do the same here?

Reply:
Thanks, @reta. Good call, added.